xplor.go

```diff
@@ -172,7 +173,7 @@ func printContents(w io.Writer, dir string, depth int) error {
 		path := filepath.Join(dir, name)
 		if info.Mode()&os.ModeSymlink != 0 {
 			if info, err = os.Stat(path); err != nil {
-				return err
+				log.Println("can't read symlink:", err)
```

Comment:
This would mean that every time we repaint while a broken symlink is visible, we'd get this error message, right? That seems a bit obnoxious. What do you think about swallowing the err in that case?

Reply:
Yeah, swallowing that is probably a good idea. I'll update the PR.